.github/workflows/changelog-assembly.yml

```diff
@@ -40,7 +42,6 @@ jobs:
             -e CITATION.cff \
             -e Cargo.lock \
             -e Cargo.toml
-          aeruginous cff-release-today CITATION.cff
```

Comment:
This application mode is deprecated.  The functionality is now part of `increment-version` which is previous instruction.